PhpAmqpLib/Wire/IO/SocketIO.php

```diff
@@ -41,9 +41,9 @@ class SocketIO extends AbstractIO
      * @param float $read_timeout
      * @param bool $keepalive
      * @param float|null $write_timeout if null defaults to read timeout
-     * @param int $heartbeat
+     * @param int $heartbeat how often to send heartbeat. 0 means off
      */
-    public function __construct($host, $port, $read_timeout, $keepalive = false, $write_timeout = null, $heartbeat)
+    public function __construct($host, $port, $read_timeout, $keepalive = false, $write_timeout = null, $heartbeat = 0)
```

Comment:
@mente it depends on the implementation. Maybe in this one it should be, and it's a good approximation. My point is that it should be enabled by default, not that a particular value is the right one.